js-exercises/abbreviate-string/abbreviateString.test.js

```diff
@@ -0,0 +1,16 @@
+import { abbreviateString } from './abbreviateString';
+
+describe('abbreviateString', () => {
+  it('abbreviates the following strings', () => {
+    expect(abbreviateString('Hacktober Fest')).toEqual('Hacktober F.');
+    expect(abbreviateString('Leeroy Fitzgerald Jenkins')).toEqual('Leeroy J.');
+    expect(abbreviateString('Some arbitrary string length here.')).toEqual('Some H.');
+  });
+
+  it('throws error on invalid parameters', () => {
+    expect(() => abbreviateString(123)).toThrow();
```

Comment:
there need to be a empty string test. You have added in code.
Imagine somebody removes from your code, and your code works fine with current tests.
But it won't do fine in world.😟

```suggestion
    expect(() => abbreviateString(123)).toThrow();
    expect(() => abbreviateString("  ")).toThrow();
```